Description: Change the section of manpages cpio.1 and mt.1 from 1L to 1.
Author: Monica Ramirez Arceda <monica@probeta.net>
Last-Update: 2010-12-16

--- a/doc/cpio.1
+++ b/doc/cpio.1
@@ -1,4 +1,4 @@
-.TH CPIO 1L \" -*- nroff -*-
+.TH CPIO 1 \" -*- nroff -*-
 .SH NAME
 cpio \- copy files to and from archives
 .SH SYNOPSIS
--- a/doc/mt.1
+++ b/doc/mt.1
@@ -1,4 +1,4 @@
-.TH MT 1L \" -*- nroff -*-
+.TH MT 1 \" -*- nroff -*-
 .SH NAME
 mt \- control magnetic tape drive operation
 .SH SYNOPSIS
